RecipesBase/test/runtests.jl

```diff
@@ -1,47 +1,47 @@
-# Run tests with `import RecipesBase` instead of `using RecipesBase` to test
-# that objects like `AbstractPlot` are properly prefixed with `RecipesBase.` in
-# the macros.
-import RecipesBase
+# Run tests with `import RecipesBase as RB` instead of `using RecipesBase`
+# or `import RecipesBase` to test that macros do not depend on the
+# namespace of the emeclosing scope.
```

Comment:
Isn't the issue that `import RecipesBase` still imports the `RecipesBase` namespace while `using RecipesBase: @recipe` does not?

Reply:
We could use `import RecipesBase: @recipe` in testing, but that would add `@recipe` to the namespace. This adds only `RB` which is an abbreviation that is not even used in the source code (afaik)